src/db/addStudentMgmt.sql

```diff
@@ -402,10 +421,17 @@ CREATE OR REPLACE FUNCTION assignMidtermGrade(student INT,
 RETURNS VOID
 AS
 $$
-BEGIN
-   RAISE WARNING 'Function not implemented';
-END
-$$ LANGUAGE plpgsql
+WITH T AS (
```

Comment:
I think it would be nice if the CTE had a more descriptive name.

Reply:
Good catch.  Fixing.